modules/storages/app/controllers/storages/admin/storages/project_storages_controller.rb

```diff
@@ -73,10 +72,24 @@ def create
     respond_with_turbo_streams(status: create_service.success? ? :ok : :unprocessable_entity)
   end
 
-  def destroy; end
+  def destroy_confirmation_dialog
+    respond_with_dialog Storages::ProjectStorages::DestroyConfirmationDialogComponent.new(
```

Comment:
🟢 Q: Do we still need an async dialog for this use case? Given the use case does not require streamable forms. A "vanilla" dialog would perhaps be simpler and more fitting

Reply:
In general I agree with you and think from the contents of the dialog it does not need to be async. However, I didn't want to change the way the `action_menu` of a table row works and this currently only works with links. So as long as there is no pressing reason why the dialog should be embedded in the page, I'd avoid that refactoring to be honest.